src/Site_Command.php

```diff
@@ -309,6 +341,210 @@ public function info( $args ) {
+	 *
+	 * [--mysql]
+	 * : Start mysql container of site.
+	 *
+	 * [--redis]
+	 * : Start redis container of site.
+	 *
+	 * [--mailhog]
+	 * : Start mailhog container of site.
+	 *
+	 * [--phpmyadmin]
+	 * : Start phpmyadmin container of site.
+	 *
+	 * [--phpredisadmin]
+	 * : Start phpredisadmin container of site.
+	 *
+	 * [--adminer]
+	 * : Start adminer container of site.
+	 *
+	 * [--anemometer]
+	 * : Start anemometer container of site.
+	 */
+	public function start( $args, $assoc_args ) {
+		$this->site_docker_compose_execute( $args[0], 'start', $args, $assoc_args);
+	}
+
+	/**
+	 * Stops containers associated with site.
+	 *
+	 * <site-name>
+	 * : Name of the site.
+	 *
+	 * [--all]
+	 * : Stop all containers of site.
+	 *
+	 * [--nginx]
+	 * : Stop nginx container of site.
+	 *
+	 * [--php]
+	 * : Stop php container of site.
+	 *
+	 * [--mysql]
+	 * : Stop mysql container of site.
+	 *
+	 * [--redis]
+	 * : Stop redis container of site.
+	 *
+	 * [--mailhog]
+	 * : Stop mailhog container of site.
+	 *
+	 * [--phpmyadmin]
+	 * : Stop phpmyadmin container of site.
+	 *
+	 * [--phpredisadmin]
+	 * : Stop phpredisadmin container of site.
+	 *
+	 * [--adminer]
+	 * : Stop adminer container of site.
+	 *
+	 * [--anemometer]
+	 * : Stop anemometer container of site.
+	 */
+	public function stop( $args, $assoc_args ) {
+		$this->site_docker_compose_execute( $args[0], 'stop', $args, $assoc_args);
+	}
+
+	/**
+	 * Restarts containers associated with site.
+	 *
+	 * <site-name>
+	 * : Name of the site.
+	 *
+	 * [--all]
+	 * : Restart all containers of site.
+	 *
+	 * [--nginx]
+	 * : Restart nginx container of site.
+	 *
+	 * [--php]
+	 * : Restart php container of site.
+	 *
+	 * [--mysql]
+	 * : Restart mysql container of site.
+	 *
+	 * [--redis]
+	 * : Restart redis container of site.
+	 *
+	 * [--mailhog]
+	 * : Restart mailhog container of site.
+	 *
+	 * [--phpmyadmin]
+	 * : Restart phpmyadmin container of site.
+	 *
+	 * [--phpredisadmin]
+	 * : Restart phpredisadmin container of site.
+	 *
+	 * [--adminer]
+	 * : Restart adminer container of site.
+	 *
+	 * [--anemometer]
+	 * : Restart anemometer container of site.
+	 */
+	public function restart( $args, $assoc_args ) {
+		$this->site_docker_compose_execute( $args[0], 'restart', $args, $assoc_args);
+	}
+
+	/**
+	 * Reload services in containers without restarting container(s) associated with site.
+	 *
+	 * <site-name>
+	 * : Name of the site.
+	 *
+	 * [--all]
+	 * : Reload all services of site(which are supported).
+	 *
+	 * [--nginx]
+	 * : Reload nginx service in container.
+	 *
+	 * [--php]
+	 * : Start php service in container.
+	 */
+	public function reload( $args, $assoc_args ) {
+		$this->site_docker_compose_execute( $args[0], 'reload', $args, $assoc_args);
+	}
+
+	private function site_docker_compose_execute( $site, $action, $args, $assoc_args ) {
+		$all = \EE\Utils\get_flag_value( $assoc_args, 'all' );
+		$no_service_specified = count( $assoc_args ) === 0 ;
```

Comment:
Maybe add a confirmation here if no assoc argument is passed to ask user that we are going to perform this operation for all. Or just add a message that we are reload/stop/restart all the services as by default.

Reply:
@rahulsprajapati How about adding the message in docblock comment itself?

Reply:
But docblock will only show message in suggestion or help right?